### PR TITLE
Add node server

### PR DESCRIPTION
**Pull request by harrypritchett**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
     "dev": "webpack-dev-server --inline --progress --open --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
+    "start": "node server.js",
     "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
     "e2e": "node test/e2e/runner.js",
     "test": "npm run unit && npm run e2e",
     "lint": "eslint --ext .js,.vue src test/unit test/e2e/specs",
-    "build": "node build/build.js"
+    "build": "node build/build.js",
+    "heroku-postbuild": "npm install --only=dev && npm run build"
```
Comment (CodingItWrong) [inline package.json]: What does only=dev accomplish here? Don't we want "non-dev" stuff to run express?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const path = require('path');
+const serveStatic = require('serve-static');
+
+let app = express();
+app.use(serveStatic(__dirname + "/dist"));
+
+const port = process.env.PORT || 5000;
```
Comment (CodingItWrong) [inline server.js]: Will this actually be accessible on port 80 to end users?

File: package.json (hunk 1 above)
Comment (harrypritchett) [reply package.json]: That happens automatically by heroku's build

File: server.js (hunk 2 above)
Comment (harrypritchett) [reply server.js]: Yes

**Comment by CodingItWrong** [comment]
Once we confirm the deployment is working, could be good to add a quick note to the readme about how it runs in production, since our team isn’t as familiar with node

**Pull request merged**